packages/graphql/src/services/questions.ts

```diff
@@ -415,3 +416,112 @@ export async function getFileUploadSas(
     fileName: blobName,
   }
 }
+
+export async function updateQuestionInstances(
+  { questionId }: { questionId: number },
+  ctx: ContextWithUser
+) {
+  // fetch the question and return null, if the question does not exist
+  const question = await ctx.prisma.question.findUnique({
+    where: {
+      id: questionId,
+    },
+    include: {
+      instances: {
+        include: {
+          sessionBlock: {
+            include: {
+              session: true,
+            },
+          },
```

Comment:
I think we still need to filter for the relevant sessions here, i.e., with status PREPARED or SCHEDULED. Otherwise the questions will be updated also in running sessions, too.